app/controllers/transactions_controller.rb

```diff
@@ -42,6 +44,25 @@ def destroy
     @transaction.destroy
   end
 
+  def copy
+    if @transaction.transfer?
+      @transaction = @transaction.dup
```

Comment:
@AlexanderUlitin I am not sure, but I think this fields will be overwritten after save.